include/ecdaa/prng.h

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ *
+ * Copyright 2017 Xaptum, Inc.
+ * 
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ * 
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License
+ *
+ *****************************************************************************/
+
+#ifndef ECDAA_RANDOM_H
+#define ECDAA_RANDOM_H
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <amcl/randapi.h>
+
+#include <stddef.h>
+
+#define AMCL_SEED_SIZE 128  // As recommended in AMCL's "rand.c"
+
+enum ecdaa_prng_initialized {
+    ECDAA_PRNG_INITIALIZED_YES,
```

Comment:
Explicitly define the numeric values and make sure the `ECDAA_PRNG_INITIALIZED_NO` is 0.  As it is right now, I think that on systems the zero new memory allocations, a new `ecdaa_prng.initialized` will default to 0, which is `ECDAA_PRNG_INITIALIZED_YES`.

Reply:
Good catch.